src/material-examples/cdk-datepicker-overview/cdk-datepicker-overview-example.ts

```diff
@@ -0,0 +1,50 @@
+import {Component} from '@angular/core';
+import {CalendarView, DateAdapter} from '@angular/cdk/datetime';
+
+
+/** @title Basic CDK datepicker */
+@Component({
+  selector: 'cdk-datepicker-overview-example',
+  templateUrl: 'cdk-datepicker-overview-example.html',
+  styleUrls: ['cdk-datepicker-overview-example.css'],
+})
+export class CdkDatepickerOverviewExample {
+  _dateSelected() {
+    console.log('Date changed');
+  }
+}
+
+
+@Component( {
+  selector: 'calendar',
+  template: `<button (click)="_firstSelected()">{{_firstDate}}</button>
+    <button (click)="_secondSelected()">{{_secondDate}}</button>
+    <div>Date: {{selected}}</div>
+  `,
+})
+export class Calendar<D> extends CalendarView<D> {
```

Comment:
maybe we can make this slightly more realistic by giving it an `@Input` list of dates to show buttons for